app/assets/stylesheets/mobile/topic-post.scss

```
@@ -323,6 +323,16 @@ blockquote {
   margin-right: 0;
 }
 
+.d-header-wrap * {
+  @include user-select(none);
+}
+
+// most importantly for android to prevent the quote menu to interfere with the text selection
+[data-identifier="post-text-selection-toolbar"].-disable-pointer-events,
+[data-identifier="post-text-selection-toolbar"].-disable-pointer-events * {
```

Does it work if we do this?

```suggestion
[data-identifier="post-text-selection-toolbar"].-disable-pointer-events {
```
If I'm understanding MDN correctly, `pointer-events:none` applies to children as long as they don't explicitly set `pointer-events: auto` 👀